packages/itwinui-css/src/input/input-flex-container.scss

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-input-flex-container {
+  --_iui-input-background-color: var(--iui-color-background);
+  --_iui-input-border-color: var(--iui-color-border-foreground);
+  --_iui-input-text-color: var(--iui-color-text);
+
+  @include iui-reset;
+  @include iui-focus($offset: -2px, $thickness: 2px);
+  width: 100%;
+  font-family: inherit;
+  font-size: var(--iui-font-size-1);
+  font-weight: var(--iui-font-weight-normal);
+  line-height: var(--iui-size-l);
+  border-radius: var(--iui-border-radius-1);
+  appearance: none;
+  height: var(--_iui-input-min-height);
+  padding-block: var(--_iui-input-padding-block);
+  padding-inline: var(--iui-size-s);
+  color: var(--_iui-input-text-color);
+  background-color: var(--_iui-input-background-color);
+  border: 1px solid var(--_iui-input-border-color);
+  transition: border-color var(--iui-duration-1) ease-out;
+  @include iui-input-size;
+
+  right: unset;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+
+  > input {
+    padding-left: var(--iui-size-s);
```

Comment:
i think padding value might be wrong (or maybe it's adding to parent's padding). doesn't match our input
![](https://user-images.githubusercontent.com/9084735/217103014-d5b89366-6ac9-4f9d-bf34-ae4c81f96287.png)
![](https://user-images.githubusercontent.com/9084735/217103079-124acdc7-08fe-4468-95ee-04bd45d93f01.png)